pkg/morph/client/netmap/client.go

```diff
@@ -80,6 +80,14 @@ func AsAlphabet() Option {
 	}
 }
 
+// TryNotary returns option to enable
+// notary invocation tries.
+func TryNotary() Option {
```

Comment:
What is `TryNotary` really? What are we trying here?

Reply:
i just rolled back how it was named before (the commit hash it attached)

Reply:
what do you suggest?

Reply:
With the current `morph` and "contract clients" probably nothing. `GoNotary`.

Reply:
in fact, noop change. did not know the low level `TryNotary` is always here and is hidden by `defaultOptions` func. dropped